tests/unit/test-cmd/test.build.js

```diff
@@ -5,11 +5,37 @@ import {it, describe} from 'mocha';
 import {assert} from 'chai';
 import sinon from 'sinon';
 
-import build, {safeFileName, FileFilter} from '../../../src/cmd/build';
+import build, {safeFileName, FileFilter,
+                getDefaultLocalizedName} from '../../../src/cmd/build';
 import {withTempDir} from '../../../src/util/temp-dir';
 import {fixturePath, makeSureItFails, ZipFile} from '../helpers';
-import {basicManifest, manifestWithoutApps} from '../test-util/test.manifest';
+import {basicManifest,
+  manifestWithoutApps,
+} from '../test-util/test.manifest';
+import {WebExtError} from '../../../src/errors';
+import {createLogger} from '../../../src/util/logger';
 
+const basicLocalizedManifest = {
+  name: '__MSG_extensionName__',
+  version: '0.0.1',
+  applications: {
+    gecko: {
+      id: 'basic-manifest@web-ext-test-suite',
+    },
+  },
+};
+
+const basicLocalizedManifestWithRepeatingPattern = {
```

Comment:
this object and also `basicLocalizedManifest` are only used once in each of their respective tests. That means they are not shared code and thus do not need to be defined at the top of the test suite file. You should define them in each test where they are needed so that someone reading the code can see everything in one place.